src/cli/printable.go

```diff
@@ -0,0 +1,56 @@
+package cli
+
+import (
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/git-town/git-town/src/util"
+)
+
+// PrintableBranchAncestry provides the branch ancestry in CLI printable format.
+func PrintableBranchAncestry(config Config) string {
```

Comment:
These functions format configuration data for printing on the CLI. They could go either into the `cli` package (as is done here) or into the `cmd` package. I tried both versions, the `cli` package feels a bit better. But I'm open to suggestions here.

Reply:
Here seems good.

Reply:
Thanks!